scripts/env/dev/init.sh

```diff
@@ -186,7 +190,12 @@ cp -f ./hosts ./hosts_local
 sed -i "s/^xx.xx.xx.xx/127.0.0.1/" ./hosts_local
 
 # Setup dev env
-ansible-playbook -i ./hosts_local --connection local setup_s3dev_centos77_8.yml -v  -k --extra-vars "s3_src=${S3_SRC_DIR}"
+if [ $ansible_automation -eq 1 ]
+then
+   ansible-playbook -i ./hosts_local --connection local setup_s3dev_centos77_8.yml -v --extra-vars "s3_src=${S3_SRC_DIR} openldappasswd=sgiammin ldapiamadminpasswd=ldapadmin"
```

Comment:
can we generate password and pass

Reply:
also the openldappasswd has wrong passwd: "sgiammin" !!! Will this fail account creation in STs ?

Reply:
done